README.md

```diff
@@ -55,6 +55,16 @@ slackutil star --include "cool-.*"
 slackutil unstar --include "uncool-.*"
 ```
 
+### Mutting/unmutting channels
+As there is no API for retrieving current `mute` status, `mute` just sends `/mute` command into channel.
+So if channel was mutted before - it will become unmutted and if it was unmutted - it will become mutted.
```

Comment:
Random 🇬🇧 drive by - 'muted', not 'mutted'. I think 'mutted' would mean to turn somebody into an ugly dog :D